ppocr/utils/save_load.py

```diff
@@ -153,26 +153,27 @@ def load_pretrained_params(model, path):
     params = paddle.load(path + '.pdparams')
 
     state_dict = model.state_dict()
-
+    
     new_state_dict = {}
     is_float16 = False
 
-    for k1 in params.keys():
-
-        if k1 not in state_dict.keys():
-            logger.warning("The pretrained params {} not in model".format(k1))
+    map_dict = MapBackboneWeights(params, state_dict)
+    
+    for k1 in map_dict.keys():
```

Comment:
The thing with map_dict is that it only load backbone params. Therefore it will only have backbone params in new_state_dict as well, which is not what we expect in loading a model.

I suggest adding params not in the backbone in the map_dict as well, but only map to themselves.